examples/speech-to-text/rest/sync/url/main.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2023-2024 Deepgram SDK contributors. All Rights Reserved.
```

Comment:
I added this updated URL example, as the older one was a bit limited in functionality. You can see what I removed below.